### PR TITLE
add 1 click enrichment

### PR DESCRIPTION
**Pull request by hps257**
Includes type detection

### CODE REVIEW

File: .ipynb_checkpoints/1_click_demo-checkpoint.ipynb [hunk 1]
```diff
@@ -0,0 +1,328 @@
+{
```
Comment (jonathanedelman) [inline .ipynb_checkpoints/1_click_demo-checkpoint.ipynb]: @hps257 what's with this file in the .ipynb_checkpints folder?

Comment (hps257) [reply .ipynb_checkpoints/1_click_demo-checkpoint.ipynb]: I was manually saving it through the icon on the notebook and it created checkpoints. It's the same copy actually

**Comment by jonathanedelman** [comment]
@hps257 this is pretty cool. unfortunately running into some issues. 

image

image

image

image


It fails because of that provider - note totally sure why. Also, I don't think we want the MICHELLE TEST provider there. Also, is there a way to display a better name for the `data-fn--840-experian-liens` one? There are also 2 providers that don't have a logo, and their name doesn't come up in the same size as the other providers - `data_gov_cpsc_loa` (that caused the failure) and `kyckr_onboarding_company_search`.

File: .ipynb_checkpoints/1_click_demo-checkpoint.ipynb (hunk 1 above)
Comment (jonathanedelman) [reply .ipynb_checkpoints/1_click_demo-checkpoint.ipynb]: @hps257 can we delete the checkpoint? no need to save it right?

**Comment by hps257** [comment]
@jonathanedelman can you use the demo account? Many providers in admin account are for testing. The issue is because the providers are not removed from provider sets completely and need to removed from provider grants manually. 

https://demyst.slack.com/archives/CCQJ5E2LA/p1562967787439200

File: .ipynb_checkpoints/1_click_demo-checkpoint.ipynb (hunk 1 above)
Comment (hps257) [reply .ipynb_checkpoints/1_click_demo-checkpoint.ipynb]: Done

**Pull request merged**